scenarios/command/README.md

```diff
@@ -273,3 +273,35 @@ python python/command_receiver.py --env-file=vehicle03.env
 ```bash
 python python/command_invoker.py --env-file=mobile-app.env
 ```
+
+### TypeScript
+
+<!-- To generate the TypeScript files to handle the protobuf payload, install [protoc](https://grpc.io/docs/protoc-installation/) compiler. The TypeScript plugin used to generate the types should already be installed from the initial setup. Note that you only need these to generate the files.
```

Comment:
we already have this instructions for C, and I believe that we also install those in the devContainer. we should unify these instructions